sdk/communication/AzureCommunicationCommon/CHANGELOG.md

```diff
@@ -1,9 +1,14 @@
 # Release History
 
-## 1.2.0 (upcoming)
+## 2.0.0-beta.1 (upcoming)
```

Comment:
I'm not convinced we should push this as a breaking change. It would require a cascade of breaking change releases. I think this was pushed by the C# folks, but I know Python is not considering this a breaking change.